lib/src/sync/exception.dart

```diff
@@ -0,0 +1,221 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+library webdriver.exception;
+
+abstract class WebDriverException implements Exception {
```

Comment:
Errors is one of the main ways that W3C differs from Selenium. See https://github.com/bazelbuild/rules_webtesting/blob/master/go/launcher/webdriver/webdriver_error.go for how I handled this in Go.